client/app/bundles/Zoon/components/WorkflowCreator.jsx

```diff
@@ -187,6 +188,21 @@ class WorkflowCreator extends React.Component {
     return this.families().length
   }
 
+  showModuleLightbox = (moduleId) => {
+    if (moduleId === null || moduleId === undefined) {
+      this.setState({showingModule: null})
+      return
+    }
+
+    const showingModule = this.props.entities.find((m) => m.id === moduleId)
+    if (showingModule === undefined) {
+      console.error(`Trying to show unexisting module with id ${moduleId}`)
```

Comment:
Yes. It's not meant to be triggered. If people observe a strange behavior, this is our way to confirm what's happening.

If we had an error collector (like airbrake or sentry), I'd sent the error there instead.